doc/source/rllib/metrics-logger.rst

```diff
@@ -181,15 +181,19 @@ to :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`:
 .. testcode::
 
     logger.log_value("some_items", value="a", reduce=None, clear_on_reduce=True)
-    logger.log_value("some_items", value="b")
-    logger.log_value("some_items", value="c")
-    logger.log_value("some_items", value="d")
+    logger.log_value("some_items", value="b", reduce=None, clear_on_reduce=True)
+    logger.log_value("some_items", value="c", reduce=None, clear_on_reduce=True)
+    logger.log_value("some_items", value="d", reduce=None, clear_on_reduce=True)
```

Comment:
The idea behind the change that makes this necessary is that...
a) There should be very few spots in the code where users log the same value (so it's not much work to write out all arguments each time we log to a give name.
b) To enforce that the metrics logger does exactly what is expected. There should be no race conditions where users use different call arguments to log values for the same stats object etc.